docs/gds-supported-platform/getting-started-gsp-local.md

```diff
@@ -148,109 +212,222 @@ You run an app by creating a [Kubernetes Deployment object](https://kubernetes.i
                   protocol: TCP
     ```
 
-1. Run the following command in the root directory to render the chart and send the output to an `output` directory:
+1. Run the following command in the root directory of the repository to render the chart and send the output to an `output` directory:
 
     ```
     mkdir output
     helm template --name prototype-kit --output-dir=output chart
     ```
 
-1. Install the contents of the `output` directory to the GSP cluster:
+1. Install the contents of the `output` directory to the local GSP instance:
 
     ```
     kubectl apply -R -f output/
     ```
 
-1. List the Deployments installed in the GSP cluster:
+    Refer to the [`kubectl apply` documentation](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply) for more information.
+
+1. List the Deployments installed in the local GSP instance:
 
     ```
     kubectl get deployments
```

Comment:
still think we need an example of the output

Reply:
ok